pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -29,6 +29,9 @@
 
 let
   debugSymbols = debug || developerBuild;
+  qtPlatform = plat: with plat;
+    if isAarch64 && isLinux then "linux-arm-generic-g++"
```

Comment:
What about armv7?

Reply:
Wow thanks for noticing. 9d0f5fd5a140c683b7a822d11c69197283782c9a